README.md

```diff
@@ -1,14 +1,7 @@
-<h5 align="center">
-  <br>
-  <a href="https://www.asyncapi.org"><img src="https://github.com/asyncapi/parser-nodejs/raw/master/assets/logo.png" alt="AsyncAPI logo" width="200"></a>
-  <br>
-  AsyncAPI CLI
-</h5>
-<p align="center">
-  <em>CLI to work with your AsyncAPI files. Currently supports validation, but it is under development for more features.</em>
-</p>
-
-<p align="center">
+<a href="https://www.asyncapi.org"><img src="./assets/logo.png" alt="AsyncAPI CLI"></a>
```

Comment:
I would prefer that it be written in markdown syntax like this since we don't need the HTML for centering the items anymore:

```suggestion
[![AsyncAPI CLI](./assets/logo.png)](https://www.asyncapi.com)
```

Reply:
Ok.

Reply:
should badges html also be replaced with markdown.

Reply:
I was going to suggest that as well, but didn't want to stray away from the goal of this PR. But if you don't mind updating the badges as well, that would be great! 😄